definitions/tools/filter_vcf_docm.cwl

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Filter variants from the DoCM detector"
+baseCommand: ["/usr/bin/perl", "/usr/bin/docm_filter.pl"]
+requirements:
+    - class: DockerRequirement
+      dockerPull: "mgibio/cle:v1.4.1"
+arguments: [
```

Comment:
$docm_vcf, $normal_cram, $tumor_cram, $output_vcf_file, $set_filter_flag is the order of arguments that docm_filter.pl takes. Just wanna make sure that "docm_filtered_variants.vcf" will be in the position after tumor_bam and before filter_docm_variants flag.

Reply:
Yep.  The negative numbers go before the `arguments` block, and positive numbers go after.